docs/profiling.md

```diff
@@ -166,6 +177,11 @@ Add the path to `libcupti.so` to the environment variable `LD_LIBRARY_PATH`.
 export LD_LIBRARY_PATH=/usr/local/cuda-10.1/extras/CUPTI/lib64/:$LD_LIBRARY_PATH
 ```
 
+If you still get the `Could not load dynamic library` message after doing this,
+check if the GPU trace shows up in the trace viewer anyway. This message
+sometimes occurs even when everything is working, since it looks for the
```

Comment:
We should fix this to at least hide the warning in the case it finds it in the second place it looks.

Reply:
Agreed. I'm not gonna gate these docs on doing that, but I'll delete this paragraph if/when we fix it. (I can take a quick look, but don't wanna spend too much time on it right now.)